### PR TITLE
Add PiNode-XMR

### PR DESCRIPTION
**Pull request by erciccione**
https://github.com/monero-ecosystem/meta/issues/37

### CODE REVIEW

File: _posts/2020-05-13-pinode-xmr.md [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+description: 'Monero Node for Single Board Computers with Web Interface and additional tools pre-configured. Self Installing.'
+introduction: 'Monero Node for Single Board Computers'
+date: 2020-05-13 00:00:00
+maintainer: 'shermand100'
+maintainer_uid: '20049380'
+main-class: 'utility'
+tags:
+- shermand100
+- raspberry
+- board
```
Comment (Lafudoci) [inline _posts/2020-05-13-pinode-xmr.md:14]: There is an existing tag "Single-Board-Computer". So I suggest following changes.

```suggestion
- Raspberry Pi
- Single-Board-Computer
```

File: _posts/2020-05-13-pinode-xmr.md [hunk 2]
```diff
@@ -0,0 +1,123 @@
+---
+layout: post
+title: 'PiNode-XMR'
+repo_name: 'PiNode-XMR'
+description: 'Monero Node for Single Board Computers with Web Interface and additional tools pre-configured. Self Installing.'
```
Comment (Lafudoci) [inline _posts/2020-05-13-pinode-xmr.md:5]: The description seems a little bit too long to me. Maybe @shermand100 could take a look to see if it fits.
```suggestion
description: 'Monero node for single board computers with web interface and pre-configured tools.'
```

Comment (erciccione) [reply _posts/2020-05-13-pinode-xmr.md:5]: I only copied the description on th GitHub page. No problem in changing it with something else.

Comment (Lafudoci) [reply _posts/2020-05-13-pinode-xmr.md:5]: OK, so it's from the author. Then I would say leave it as it is. But I still hope the format could be revised to only capital for first letter to make the webpage more consistent.

**Comment by erciccione** [comment]
Added all suggestions

**Comment by Lafudoci** [comment]
@erciccione, I just realized I confused about "introduction" and "description". The one that needs a shorter sentence is the introduction. And you have already done it correctly in the first commit. Thanks for your great work, my bad.

**Pull request merged**